lib/phx_tools_web/live/phx_tools_components.ex

```diff
@@ -0,0 +1,268 @@
+              </p>
+            </div>
+            <div
+              class="mx-auto"
+              data-asciicast={
+                if @live_action == :macOS,
+                  do: "bJMOlPe5F4mFLY0Rl6fiJSOp3",
+                  else: "XhDpRstBJ4df2gfiRfp0awDPO"
+              }
+              id={"asciinema-#{Utils.random_id()}"}
+              phx-hook="AsciinemaHook"
+            >
+            </div>
+
+            <p class="md:my-5 text-white md:px-12 text-xs md:text-sm lg:text-base font-martian">
+              Read about website updates here -
+              <a
+                href="https://optimum.ba/blog/exciting-updates-to-phx-tools"
+                class="text-[#24B2FF] font-martian text-sm underline"
+                target="_blank"
+              >
+                https://optimum.ba/blog/exciting-updates-to-phx-tools
+              </a>
+            </p>
+          </div>
+        </div>
+      </div>
+      <.footer />
+    </div>
+    """
+  end
+
+  @spec os_instructions(assigns()) :: rendered()
+  def os_instructions(assigns) do
+    ~H"""
+    <div
+      id="installation-instructions"
+      phx-click-away={JS.hide(to: "#installation-instructions")}
+      class="hidden font-normal solved-height font-martian bg-[#110A33] absolute md:w-[90%]"
+    >
+      <div class="block">
+        <div class="h-full shadow-[#C2B8FF] shadow-md rounded-md pb-2">
+          <div class="text-start px-[3%] lg:text-xl md:text-lg sm:text-md">
+            <h1 class="text-white text-center text-sm md:text-base lg:text-xl lg:my-[5%] md:my-[2%] sm:my-[2%] lg:pt-5">
+              <%= @live_action %> installation process
+            </h1>
+            <ol class="list-decimal ml-3 pl-5 text-xs md:text-sm lg:text-base text-white lg:mt-4 sm:mt-2 leading-6">
+              <%= for instruction <- render_instructions(@live_action) do %>
+                <li class="mb-2"><%= raw(instruction) %></li>
+              <% end %>
+            </ol>
+          </div>
+        </div>
+      </div>
+    </div>
+    """
+  end
+
+  attr :id, :string, required: true
+  attr :class, :string
+  attr :href, :string, required: true
+  attr :os_name, :string, required: true
+
+  @spec os_link_button(assigns()) :: rendered()
+  def os_link_button(assigns) do
+    ~H"""
+    <.link href={@href}>
+      <div
+        id={@id}
+        class={[
+          "border-2 border-[#755FFF] py-2 rounded-xl cursor-pointer hover:bg-indigo-850 flex items-center md:w-44 space-x-2",
+          @class
+        ]}
+      >
+        <div class="bg-white w-6 h-6 rounded-full flex items-center justify-center">
+          <Icons.os_icon os_name={@os_name} />
+        </div>
+        <h1 class="text-center text-white text-sm md:text-base font-martian"><%= @os_name %></h1>
+      </div>
+    </.link>
+    """
+  end
+
+  defp render_instructions(live_action) do
+    case live_action do
```

Comment:
We can do the pattern matching in the function head, i.e. `defp render_instructions(:Linux) do`